FirebaseABTesting/Sources/Public/FIRExperimentController.h

```diff
@@ -17,15 +17,15 @@
 @class ABTExperimentPayload;
 
 // Forward declaration to avoid importing into the module header
-typedef NS_ENUM(int32_t, ABTExperimentPayload_ExperimentOverflowPolicy);
```

Comment:
Changing the type name will break RemoteConfig users who lock their version and do a `pod update`. Can we keep the type name unchanged until the next major version.

Otherwise, we could consider bumping the major version since we don't really have a public API for ABT.

cc: @ryanwilson

Reply:
I'd prefer bumping the major version since the underscores aren't idiomatic Objective-C. What do you guys think?